tests/publiccloud/pcw_client.pm

```diff
@@ -0,0 +1,37 @@
+use base "consoletest";
+use serial_terminal 'select_serial_terminal';
+use strict;
+use warnings;
+use testapi;
+use mmapi;
+use utils qw(zypper_call);
+use network_utils qw(setup_static_network);
+use lockapi;
+
+sub run {
+    my ($self, $args) = @_;
+    select_serial_terminal;
+
+    record_info('system', script_output('cat /etc/os-release'));
+    setup_static_network(ip => '10.0.2.15/15', gw => '10.0.2.2', iface => 'eth0');
+    assert_script_run('echo "10.0.2.20  microos" >> /etc/hosts');
+    zypper_call('in -y iputils git');
+
+    assert_script_run('mkdir /root/.ssh');
+    assert_script_run('curl -f -v ' . autoinst_url . '/data/slenkins/ssh/id_rsa > /root/.ssh/id_rsa');
```

Comment:
Please generate new keys and put them into `data/publiccloud/pcw/`.

Reply:
I still don't understand how the systems are installed. If there is some 'normal user' you can SCP the public key via password authorized SSH.